tests/Functional/FunctionalTestCase.php

```diff
@@ -4,71 +4,40 @@
 
 namespace tests\Libero\ContentApiBundle\Functional;
 
-use DirectoryIterator;
-use InvalidArgumentException;
-use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\KernelInterface;
 use tests\Libero\ContentApiBundle\Functional\App\Kernel;
+use function tests\Libero\ContentApiBundle\capture_output;
 
-abstract class FunctionalTestCase extends TestCase
+abstract class FunctionalTestCase extends KernelTestCase
```

Comment:
State (such as `InMemoryItems`) wasn't be reset. This makes it slower; will see if there's a way to reset the kernel/container and extract the change.